webpack/assets/javascripts/react_app/components/users/PersonalAccessTokens/PersonalAccessTokens.js

```diff
@@ -0,0 +1,65 @@
+import React, { Fragment, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import NewPersonalAccessToken from './NewPersonalAccessToken';
+import PersonalAccessTokenForm from './PersonalAccessTokenForm';
+import PersonalAccessTokensList from './PersonalAccessTokensList';
+import { translate as __ } from '../../../common/I18n';
+
+const PersonalAccessTokens = ({
+  url,
+  canCreate,
+  tokens,
+  newPersonalAccessToken,
+  clearNewPersonalAccessToken,
+  getPersonalAccessTokens,
+  revokePersonalAccessToken: revokePersonalAccessTokenAction,
+}) => {
+  useEffect(() => {
+    getPersonalAccessTokens({ url });
+  }, []);
```

Comment:
why using  empty`[]` ? it makes the `useEffect` not being affected by `url` prop changes.
```suggestion
  useEffect(() => {
    getPersonalAccessTokens({ url });
  }, [url]);
```

Reply:
@amirfefer It's probably from my [comment above](https://github.com/theforeman/foreman/pull/7806#discussion_r452321010), sorry :smile: 

I agree with adding `url` to the dependency array